### PR TITLE
AWS Batch single queue only

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://nextflow.slack.com/archives/C02T97HAV5M/p1721331937919219?thread_ts=1721331254.292239&cid=C02T97HAV5M

Initial pass at this.

cc @robsyme

### CODE REVIEW

**Review by robsyme — APPROVED** [review]
Perfect. Thanks Jason!

**Comment by ewels** [comment]
Might be nice to make this more specific to being one queue _per process_. A single pipeline run can support multiple queues: eg. Process A goes to queue A, Process B goes to queue B..

File: docs/aws.md [hunk 1]
```diff
@@ -214,6 +214,8 @@ aws {
 
 Different queues bound to the same or different Compute Environments can be configured according to each process' requirements.
 
+Nextflow on AWS Batch supports only a single queue in your pipeline configuration. Multiple queues are not supported.
+
```
Comment (pditommaso) [inline docs/aws.md:218]:
```suggestion
```

We need to correct that's possible to provide multiple queues in the same settings, that's a very specific exception

File: docs/aws.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ The `aws` command can be made available by either (1) installing it in the conta
 To configure your pipeline for AWS Batch:
 
 1. Specify the AWS Batch {ref}`executor <awsbatch-executor>`
-2. Specify one or more AWS Batch queues with the {ref}`process-queue` directive
+2. Specify an AWS Batch queue with the {ref}`process-queue` directive
```
Comment (pditommaso) [inline docs/aws.md:193]: Note that also below is mentioned that multiple queue can be used

**Comment by pditommaso** [comment]
Closing in favour of https://github.com/nextflow-io/nextflow/pull/5154

**Pull request closed without merging**